prediction_market_agent/agents/safe_guard_agent/api.py

```diff
@@ -0,0 +1,66 @@
+import fastapi
```

Comment:
Not great to have this API in PMA, but I think the whole Safe Guard agent should have its own repository as it isn't for for PMA. So just keeping everything together so we can cut it out later at one piece.